### DISCUSSION TITLE
Welcome to poolifier Discussions!

### DISCUSSION THREAD

**Discussion by pioardi** [poolifier/poolifier#68] [General]
## 👋 Welcome to poolifier discussions!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rubiin** [comment]
Can we make multiple workers syntax more simple. Currently we have to wrap it around a function and use if/else logic
